src/main/java/seedu/address/logic/commands/TagCommand.java

```diff
@@ -29,7 +29,7 @@ public class TagCommand extends Command {
             + "Existing tags will not be overwritten by the input.\n"
             + "At least one tag must be present.\n"
             + "Parameters: INDEX (must be a positive integer) "
-            + PREFIX_TAG + "[tag]...\n"
+            + PREFIX_TAG + "[TAG]...\n"
```

Comment:
[] means optional. Should it be on the whole "t/TAG"? (technically from my code)

Reply:
Yep I agree with Wei Ming